engine/admin/logging/journald.md

```diff
@@ -87,4 +87,13 @@ logs:
     reader.add_match('CONTAINER_NAME=web')
 
     for msg in reader:
-      print '{CONTAINER_ID_FULL}: {MESSAGE}'.format(**msg)
\ No newline at end of file
+      print '{CONTAINER_ID_FULL}: {MESSAGE}'.format(**msg)
+
+## Journald configuration
+
+Docker hosts with many containers may produce massive amounts of log-messages.
+The default configuration of journald is to limit the number of messages stored per service per time-unit.
```

Comment:
Please put back-ticks around `journald`. Consider rewriting to something like "By default, `journald` limits the number of messages stored per service per time-unit."